### PR TITLE
NEXT: Docs content expansion

### PR DESCRIPTION
**Pull request by endigo9740**
## Linked Issue

#2399

## Description

- [x] Updates and improves content for several pages
- [x] Updated homepage with "early access" warning
- [x] Added Design System > Themes section
- [x] First draft of theme enhancements

### CODE REVIEW

**Comment by endigo9740** [comment]
Still remaining:

- [ ] Header redesign
  - [ ] Drop h1 text
  - [ ] Add Docs | Theme links
  - [ ] Add search bar
- [ ] New Sections
  - [ ] Installation (placeholder)
  - [ ] Styling
  - [ ] Dark Mode
- [ ] Finalize:
  - [ ] Move "Utilities" under Docs
- [ ] New components
 - [ ] Onboarding Steps

File: sites/next.skeleton.dev/src/content/docs/installation/nextjs.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        Start by creating a new project using `create-next-app`:
+        ```console
+        npx create-next-app@latest my-app --typescript --tailwind --eslint
+        ```
+    </ProcessStep>
+    <ProcessStep step="2">
+        ## Install Skeleton
+        Install the Skeleton core and Skeleton React packages.
+        ```console
+        npm i -D @skeletonlabs/skeleton @skeletonlabs/skeleton-react
```
Comment (Sarenor) [inline sites/next.skeleton.dev/src/content/docs/installation/nextjs.mdx:27]: Do we want to append an ``@next`` tag to this for now or is it just intended as an "installation will work like this but currently doesn't" page?

File: sites/next.skeleton.dev/src/content/docs/introduction.mdx [hunk 2]
```diff
@@ -16,6 +18,52 @@ export const components = componentSet;
...
+
+Skeleton provides access to a powerful [Figma UI Kit](/docs/figma) to assist designers in drafting a visual concept of your upcoming projects.
+
+### Tailwind++
+
+Skeleton fully embraces [Tailwind](https://tailwindcss.com/) and its powerful utility-based approach to styling, built around the [Atomic Design Methodology](https://atomicdesign.bradfrost.com/chapter-2/). While also extending the capabilities in really meaningful ways, including not only themes but comprehensive systems for colors, typography, spacing, and more.
+
+### Theme System
+
+Skeleton provides best in class support for themes and styling, including a full set of fully curated themes built-in, as well as a powerful but friendly [theme generator](https://themes.skeleton.dev/) for creating new themes that fully adhere to your unique branding and design.
```
Comment (Sarenor) [inline sites/next.skeleton.dev/src/content/docs/introduction.mdx:37]: Maybe ``easy to use`` instead of ``friendly``? I'm not sure myself.

**Review by Sarenor — APPROVED** [review]

**Comment by TazorDE** [comment]
Not sure if this is included in this PR but I noticed a small typo on the /tailwind/forms page on my full docs readthrough
image

File: sites/next.skeleton.dev/src/content/docs/installation/nextjs.mdx (hunk 1 above)
Comment (endigo9740) [reply sites/next.skeleton.dev/src/content/docs/installation/nextjs.mdx:27]: Good call. I'll add this.

**Pull request merged**